tests/cypress/e2e/unit_tests/p1_2_fleet.spec.ts

```diff
@@ -1246,3 +1246,58 @@ describe('Test non-yaml file into bundle.', { tags: '@p1_2'}, () => {
     })
   )
 });
+
+describe('Test Fleet doNotDeploy.', { tags: '@p1_2'}, () => {
```

Comment:
Consider quotes on doNotDeploy.
Consider adding option right after

Reply:
Actually, I think the test title should explain in simple words what is meant to do.

If you say 'Test Fleet with "doNotDeploy" option set to "true" is unable to ....' I am fine with it.
but simply saying 'Test Fleet with "doNotDeploy: true"' I think it is incomplete.